### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by twz123**

### CODE REVIEW

**Comment by jstr-zal** [comment]
+1

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#/usr/bin/env sh
+
+set -e
+
+. "${0%/*}/build-vars" || { echo "build-vars not found"; exit 1; }
+
+docker build --no-cache -t $DOCKER_TAG "${0%/*}"
```
Comment (cberg-zalando) [inline build.sh]: If someone uses a version containing spaces, which someone should punish,  the docker tag will contain spaces and the command will fail.

What about putting spaces around, like you do with the argument afterwards?

File: MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1 @@
+Tom Wieczorek <tom.wieczorek@zalando.de>
```
Comment (nabcos) [inline MAINTAINERS]: Maybe add the team address as well?

File: build.sh (hunk 1 above)
Comment (twz123) [inline build.sh]: I don't know if it's even possible to have spaces inside a Docker tag. Docker has very strong requirements for its tags. I didn't use quotes to have a "cleaner" look ;) Spaces in directory names are more likely, so I added the quotes for the second parameter.

Comment (nabcos) [inline build.sh]: I would suspect a docker tag with whitespace is invalid anyway.

Comment (cberg-zalando) [inline build.sh]: Yes. But assume, that I use "lat est" as the version, then the resulting docker call will look like this:

```
docker build --no-cache -t os-registry.stups.zalan.do/phrs/openjdk:lat est .
```

But I guess, we can safely assume, that the user of this script is not a normal user and should know what he does… Hopefully.

File: MAINTAINERS (hunk 2 above)
Comment (twz123) [inline MAINTAINERS]: I thought this is about individuals, but I can add the team, of course.

**Comment by nabcos** [comment]
+1

**Pull request merged**